src/processTargets/modifiers/SmallPaintStage.ts

```diff
@@ -0,0 +1,66 @@
+import { zip } from "lodash";
+import { Target } from "../../typings/target.types";
+import {
+  ContainingScopeModifier,
+  EveryScopeModifier,
+} from "../../typings/targetDescriptor.types";
+import { ProcessedTargetsContext } from "../../typings/Types";
+import { ModifierStage } from "../PipelineStages.types";
+import { TokenTarget } from "../targets";
+import { NonWhitespaceSequenceStage } from "./scopeTypeStages/RegexStage";
+import { InteriorOnlyStage } from "./InteriorStage";
+
+export type NonWhitespaceOrQuoteSequenceModifier = (
+  | ContainingScopeModifier
+  | EveryScopeModifier
+) & {
+  scopeType: { type: "nonWhitespaceOrQuoteSequence" };
+};
+
+/**
+ * Intersection of NonWhitespaceSequenceStage and InteriorOnlyStage
+ * Expand the target until reaching a white space or surrounding pair.
+ * If there is no surrounding pair defaults to the non white space sequence
+ */
+export default class SmallPaintStage implements ModifierStage {
+  constructor(private modifier: NonWhitespaceOrQuoteSequenceModifier) {}
+
+  run(context: ProcessedTargetsContext, target: Target): Target[] {
+    const paintStage = new NonWhitespaceSequenceStage({
+      type: "containingScope",
```

Comment:
We don't just want containing scope. We should pass in the same type as used for the small paint modifier. ie if the user says "take every small paint" we want the call to `NonWhitespaceSequenceStage` to be every as well

Reply:
Hmm not sure.  I'd be tempted to handle "every" specially for this modifier.  When target is empty, I'd expand to nearest containing "inside", and then run "every paint" on that.  Or maybe that's what you're suggesting?

Tbh I'm not really sure what to do with "every" when target is not empty 🤔

Reply:
Every will be solved by just passing every to the paint modifier. Then running all the target through an intersection. Basically you will get every paint within the matching pair. If we have no surrounding pair paint and short paint becomes equivalent.

Reply:
Is the solution that I've gone with and I think the way works is pretty intuitive.